include/jemalloc/internal/tsd.h

```diff
@@ -59,6 +59,7 @@ typedef ql_elm(tsd_t) tsd_link_t;
 #define TSD_DATA_SLOW							\
     O(tcache_enabled,		bool,			bool)		\
     O(reentrancy_level,		int8_t,			int8_t)		\
+    O(min_init_state_nfetched,		uint8_t,	uint8_t)	\
```

Comment:
@guangli-dai this part just got added (see David's comments), in case you want to take another look.

Reply:
As discussed offline, it looks good to me. Although it's rarely possible, I will open up an issue to record the potential leak caused by the coincide of the thread shut down and the num deallocs crossing the threshold.